src/app/services/query.service.ts

```diff
@@ -128,7 +128,7 @@ export class QueryService {
   private _treeNodeCountsUpdate: boolean;
   private _exportFormats: object[] = [];
   private _isLoadingExportFormats = false;
-  private _exportDataView = 'default';
+  private _exportDataView = 'surveyTable'; // NTR specific
```

Comment:
do we keep the NTR specific export data view format, or use 'default'?

Reply:
Maybe keep it as 'default', since down below the _exportDataView value gets updated via config service